Zach-Trani/markdown-editor
# fix: abstracting bootstrap icons svg's to separate components

closes #12 

# Changes
* Create new components for the bootstrap icon svg's

# Testing
* Verify bootstrap icons render and look the same
* Verify trashcan icon still clears the markDownText
